particles/Tutorial/Javascript/3_RenderSlots/RenderSlots.arcs

```diff
@@ -6,18 +6,18 @@ particle ParentParticle in 'parent.js'
   // This particle renders to the root slot ("consumes" it), and provides a slot inside it called "mySlot" in which another particle can render
   // itself. The child particle will be rendered inside a special div with the identifier "mySlot", which this particle will need to provide in
   // its HTML.
-  consume root
-    provide mySlot
+  root: consumes Slot
+    mySlot: provides
 
 // The "child" particle. Instead of consuming "root" it consumes "mySlot", which connects it to the slot provided by ParentParticle.
 particle ChildParticle in 'child.js'
-  consume mySlot
+  mySlot: consumes
 
 recipe RenderSlotsRecipe
   ParentParticle
-    consume root
-      provide mySlot as slot0
+    root: consumes
+      mySlot: provides theSlot
```

Comment:
Is there a more meaningful name than `theSlot`? Again, asking from a best practice, and most illustrative/explanatory to a new Arcs reader, perspective. @Cypher1 @piotrswigon

Reply:
childSlot? In the real world the recipe would have a use case and that would suggest a better name.